_changelog/entries/2021/2021-03-15-zoom-v1-fixed-schema-report-meetings-webinars.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Zoom (v1) update: Fixed schema for `report_meetings` and `webinars`
+content-type: "changelog-entry"
+date: 2021-03-15
+entry-type: improvement
+entry-category: "integration"
+connection-id: "zoom"
+connection-version: "1"
+---
+
+{{ site.data.changelog.metadata.single-integration | flatify }}
+
+Schema fixed for {{ this-connection.display_name }}'s `report_meetings` and `webinars`.
```

Comment:
1. What exactly was the change here and why was it necessary?
2. Make sure you change `webinars` to `report_webinars`, as this is the applicable table for this PR.
2. What was changed in the `report_webinars` table?